chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -290,15 +290,17 @@ We will rely on *likelihood maximization*, the very same concept that we encount
 ### Log-Likelihood
 -->
 
-### *dịch tiêu đề phía trên*
+### Log hợp lý (Log-likelihood)
 
 <!--
 The softmax function gives us a vector $\hat{\mathbf{y}}$, which we can interpret as estimated conditional probabilities of each class given the input $x$, 
 e.g., $\hat{y}_1$ = $\hat{P}(y=\mathrm{cat} \mid \mathbf{x})$.
 We can compare the estimates with reality by checking how probable the *actual* classes are according to our model, given the features.
 -->
 
-*dịch đoạn phía trên*
+Hàm softmax đưa cho chúng ta một vector $\hat{\mathbf{y}}$, thứ có thể được diễn giải như là những xác suất có điểu kiện của từng phân loại được ước lượng dựa trên những đặc trưng đã biết.
+Ví dụ: $\hat{y}_1$ = $\hat{P}(y=\mathrm{cat} \mid \mathbf{x})$.
+Chúng ta có thể so sánh sự ước lượng với thực tế bằng cách kiểm tra độ chắn chắn của những phân loai thực tế dựa trên mô hình của chúng ta với những đặc trưng đã biết.
```

Comment:
```suggestion
Chúng ta có thể so sánh sự ước lượng với thực tế bằng cách kiểm tra độ chắc chắn của mô hình với các lớp *thực sự* khi được cho trước các đặc trưng.
```

Reply:
`sự ước lượng` nghe nó kỳ sao ấy? Đó là đang chỉ quá trình (cách ước lượng) hay kết quả (giá trị ước lượng)?

```suggestion
Để biết các ước lượng có sát với thực tế hay không ta tính _khả năng_ mà nhãn thật sự giống với dự đoán của mô hình của chúng ta khi cho trước các đặc trưng.
```

Reply:
yeah nó là `giá trị ước lượng`. Vế sau thì mình cảm giác vẫn hơi lủng củng, mình để xuất như sau
```suggestion
Để biết các ước lượng có sát với thực tế hay không, ta kiểm tra xác suất mà mô hình gán cho lớp *thật sự* biết các đặc trưng.
```

Reply:
@tanduong btw bạn có thể để lại email để mình mời vào slack thảo luận không? Mình sẽ xóa comment ngay sau đó.

Reply:
dntan09 at gmail nhe bạn.

Reply:
@tanduong oh có ai đó đã mời bạn rồi, bạn thử xem email xem có invite của slack không nhé.

Reply:
Câu gốc là thế này:

```
We can compare the estimates with reality by checking how probable the *actual* classes are according to our model, given the features.
```

`according to our model, given the features.` là một cụm, nghĩa là cho feature và dùng model đưa ra dự đoán.
Còn phần trước đó là nếu ta muốn biết dự đoán có đúng hay không thì phải tính khả năng (mình không chắc có nên dịch chữ probable ra xác suất hay không) dự đoán là đúng. Để biết là đúng thì phải so với nhãn thật.

Mọi người xem có thể viết lại gọn hơn hông nhe :))

Reply:
mình thấy dịch vậy là hợp lý, có vẻ tác giả đang ám chỉ xác suất có điều kiện `P(prediction == true class | features)`

Reply:
> có vẻ tác giả đang ám chỉ xác suất có điều kiện `P(prediction == true class | features)`
uh.